locales/en.yml

```diff
@@ -53,6 +54,22 @@ en:
                 --script-readable  display information in a script readable format
                 -h, --help         print this help
 
+        restart: |-
+          Restarts the given systemd service in the box
+
+          Usage: vagrant service-manager restart <object> [options]
+
+          Object:
+              Object can be any available service configured in the box using systemd unit files
+
```

Comment:
Object is a service provided via sccli or systemd.  For example:
docker, kubernetes (or k8s), or openshift.